packages/serialization/form/test/common/formSerializationWriter.ts

```diff
@@ -33,6 +34,8 @@ describe("FormSerializationWriter", () => {
     const formContent = formSerializationWriter.getSerializedContent();
     const form = new TextDecoder().decode(formContent);
 
+    console.log("\n\n-----------------" + form + "-----------------\n\n");
```

Comment:
This is not needed yes?
```suggestion
```

Reply:
> This is not needed yes?

Yep I forgot that one, thanks for catching it